Gemfile

```diff
@@ -44,7 +44,7 @@ gem 'tzinfo-data', platforms: %i[windows jruby]
 
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍